src/utils/event-manager/event-manager.js

```diff
@@ -107,27 +107,32 @@ export default class EventManager {
     }
   }
 
+  /*
+   * Enable/disable recognizer for the given event
+   */
+  _enableRecognizer(name, enabled) {
```

Comment:
nit: `_toggleRecognizer` might be a better name since this is used for both enabling/disabling